src/blocks/catalog.scss

```diff
@@ -0,0 +1,22 @@
+
+$cardwidth: 200px;
+$gapX: 48px;
+$gapY: 46px;
+$cardCountInRow: 4;
```

Comment:
Move variables to separate file

Reply:
I know this is how it should be done, but when I attach a separate file, they stop working, maybe I'll come to QnA because I don't know what I'm doing wrong anymore😅